Changing title for internal installations

mayankkapoor opened the pull request:
This will handle the use case for internal assemble installations. Let
me know if there is a better way to handle the use case of showing a
different name in title if internally deployed.

mayankkapoor committed:
Changing title for internal installations

This will handle the use case for internal assemble installations. Let
me know if there is a better way to handle the use case of showing a
different name in title if internally deployed.

mayankkapoor committed:
fix typos

sorenh commented:
Use Django's settings system. Add a setting called e.g. AASEMBLE_OVERRIDE_NAME

Pass the name into the template context using something like getattr(settings, 'AASEMBLE_OVERRIDE_NAME', 'aaSemble')

Use a context processor to make sure it's always passed in. Much simpler than having to pass it in explicitly to every `render` call.

mayankkapoor committed:
Moved site name change to settings

Now we’re using a context processor to apply the context keyword
“title” to display the name of the site.

sorenisanerd commented on aasemble/django/apps/main/context_processors.py:
```diff
@@ -0,0 +1,11 @@
+    try:
+        title = settings.AASEMBLE_OVERRIDE_NAME
+    except AttributeError:
+        title = 'aaSemble'
```
`title = getattr(settings, 'AASEMBLE_OVERRIDE_NAME', 'aaSemble')` should do the trick. I.e. without the `default=`.

mayankkapoor committed:
Now using getattr
